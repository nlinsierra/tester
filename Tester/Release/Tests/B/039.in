2H KH 4C 2S AS 3H 5C AC
