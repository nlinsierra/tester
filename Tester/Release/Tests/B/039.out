44
2C 3C 6C 7C 8C 9C TC JC QC KC 2D 3D 4D 5D 6D 7D 8D 9D TD JD QD KD AD 4H 5H 6H 7H 8H 9H TH JH QH AH 3S 4S 5S 6S 7S 8S 9S TS JS QS KS
